openpype/tools/utils/lib.py

```diff
@@ -111,13 +113,23 @@ def get_qta_icon_by_name_and_color(icon_name, icon_color):
         variants.append("{0}.{1}".format(key, icon_name))
 
     icon = None
+    used_variant = None
     for variant in variants:
         try:
             icon = qtawesome.icon(variant, color=icon_color)
+            used_variant = variant
             break
         except Exception:
             pass
 
+    if used_variant is None:
+        log.info("Didn't find icon \"{}\"".format(icon_name))
+
+    elif used_variant != icon_name:
+        log.info("Icon \"{}\" was not found \"{}\" is used instead".format(
+            icon_name, used_variant
+        ))
```

Comment:
Wouldn't this happen quite often? Anywhere where not explicitly `fa.` is prepended to the icon name for example?

E.g. [here](https://github.com/pypeclub/OpenPype/blob/57feaee44b8bff1eb7156b25bad70c978b715fec/openpype/tools/utils/lib.py#L155-L156)?

Should this be `log.debug`? And still be prepared for quite some logging?

Reply:
It is called once for each icon.